packages/world/src/World.js

```diff
@@ -1,3 +1,4 @@
+// @flow
```

Comment:
Unfortunately it appears like something in my tool chain expects the flow comment to be on the first line. I don't know if it's flow, eslint, or IDE plugin yet.